### ISSUE TITLE
WMS 1.3 GetFeatureInfo

### ISSUE THREAD

**Issue by zoran995** [CesiumGS/cesium#9021]
Cesium currently has implemented GetFeatureInfo for WMS which works only with WMS 1.1.0 and not with 1.3.0 specification of service. According to the implementation specification, in WMS 1.3.0 identifier of pixel coordinates are 'i' and 'j' instead of 'x' and 'y' as it was in WMS 1.1.0. 

In source code this part is [here](https://github.com/CesiumGS/cesium/blob/master/Source/Scene/WebMapServiceImageryProvider.js#L256-L261). This could probably be updated to (I am already using it like this)

```
var pickFeatureParams = {
  query_layers: options.layers,
  info_format: "{format}",
};
// use correct pixel coordinate identifier based on version
if (parseFloat(resource.queryParameters.version) >= 1.3) {
  pickFeatureParams.i = '{i}';
  pickFeatureParams.j = '{j}';
} else {
  pickFeatureParams.x = '{i}';
  pickFeatureParams.y = '{j}';
}
```

**Comment by OmarShehata** [comment]
Thanks for reporting @zoran995 ! Do you have a WMS 1.3 imagery provider you can share that you're using to test? If you'd like to open a pull request for this we'd be happy to review, see contributing guide here: https://github.com/CesiumGS/cesium/blob/master/CONTRIBUTING.md#opening-a-pull-request

**Comment by zoran995** [comment]
Currently don't have a public version of the application available (a server that is running it is in an isolated environment). The application that we are using is TerriaJS.
The WMS service that I used for testing is Luxembourg WMS of [administrative units](https://wms.inspire.geoportail.lu/geoserver/au/wms?SERVICE=WMS&service=WMS&version=1.3.0&request=GetCapabilities), link picked up from their metadata catalogue. Most of the INSPIRE services are published using WMS 1.3.0.

Request sent from the application is 
```
https://wms.inspire.geoportail.lu/geoserver/au/wms?SERVICE=WMS&version=1.3.0&service=WMS&request=GetFeatureInfo&layers=AU.AdministrativeUnit&bbox=626172.1357121654%2C6261721.357121639%2C782715.1696402058%2C6418264.391049679&width=256&height=256&crs=EPSG%3A3857&query_layers=AU.AdministrativeUnit&x=98&y=29&info_format=application%2Fjson
```

while the correct request for GetFeatureInfo is
```
https://wms.inspire.geoportail.lu/geoserver/au/wms?SERVICE=WMS&version=1.3.0&service=WMS&request=GetFeatureInfo&layers=AU.AdministrativeUnit&bbox=626172.1357121654%2C6261721.357121639%2C782715.1696402058%2C6418264.391049679&width=256&height=256&crs=EPSG%3A3857&query_layers=AU.AdministrativeUnit&i=98&j=29&info_format=application%2Fjson
```

**Comment by bampakoa** [comment]
@zoran995 @OmarShehata would you need some help this?
